pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -26,7 +26,9 @@
     <PrereleaseLibraryPackage Include="System.Configuration.ConfigurationManager" />
     <PrereleaseLibraryPackage Include="System.Data.Odbc" />
     <PrereleaseLibraryPackage Include="System.Data.DataSetExtensions" />
-    <PrereleaseLibraryPackage Include="System.Data.SqlClient" />
+    <PrereleaseLibraryPackage Include="System.Data.SqlClient">
```

Comment:
Are these the only 2 serviced packages? Also is there a reason you needed to bump their dependencies?

Reply:
Yes those where the only 2 serviced packages in the compat pack closure. Well don't we want the compat pack to point to the latest packages?

Reply:
If there are particular fixes that are needed for the compat pack then yes. Otherwise this is just going to be inconsistent and unnecessary to try and maintain against the latest serviced packages. If an application opts into the serviced package then they will reference it directly and will lift the version. 

That said I'm OK updating the full set from time-to-time but I just don't want to force us to keep this updated whenever we do a servicing.

Reply:
Ok, sounds reasonable. Does it makes sense to update it now that I'm touching the compat pack? Or Should I just revert this changes?

Reply:
I honestly would avoid updating them until we need a fix as part of the compat pack specifically.

Reply:
Fixed.